docs/site/Controller-generator.md

```diff
@@ -106,43 +114,96 @@ export class TodoController {
     @repository(TodoRepository) public todoRepository: TodoRepository,
   ) {}
 
-  @post('/todos')
-  async create(@requestBody() obj: Todo): Promise<Todo> {
-    return await this.todoRepository.create(obj);
+  @post('/todos', {
+    responses: {
+      '200': {
+        description: 'Todo model instance',
+        content: {'application/json': {'x-ts-type': Todo}},
+      },
+    },
+  })
+  async create(@requestBody() data: Todo): Promise<Todo> {
+    return await this.todoRepository.create(data);
   }
 
-  @get('/todos/count')
+  @get('/todos/count', {
+    responses: {
+      '200': {
+        description: 'Todo model count',
+        content: {'application/json': {'x-ts-type': Number}},
+      },
+    },
+  })
   async count(@param.query.string('where') where?: Where): Promise<number> {
     return await this.todoRepository.count(where);
   }
 
-  @get('/todos')
-  async find(@param.query.string('filter') filter?: Filter): Promise<Todo[]> {
+  @get('/todos', {
+    responses: {
+      '200': {
+        description: 'Array of Todo model instances',
+        content: {
+          'application/json': {
+            schema: {type: 'array', items: {'x-ts-type': Todo}},
+          },
+        },
+      },
+    },
+  })
+  async find(
+    @param.query.object('filter', getFilterSchemaFor(Todo)) filter?: Filter,
+  ): Promise<Todo[]> {
     return await this.todoRepository.find(filter);
   }
 
-  @patch('/todos')
+  @patch('/todos', {
+    responses: {
+      '200': {
+        description: 'Todo PATCH success count',
+        content: {'application/json': {'x-ts-type': Number}},
+      },
+    },
+  })
   async updateAll(
-    @requestBody() obj: Todo,
+    @requestBody() data: Todo,
     @param.query.string('where') where?: Where,
```

Comment:
shouldn't `where` an object instead of string?

Reply:
> shouldn't `where` an object instead of string?

Good point, will fix.